### PR TITLE
Add macOS support to snapshot

### PR DESCRIPTION
**Pull request by ohayon**
This is a continuation/cleanup of https://github.com/fastlane/fastlane/pull/7363 to get it pushed through after a while of marinating. I know that macOS support is not something we are prioritizing, but it felt like this does not add much complexity and is a nice-to-have. 

I am definitely open to this being kiboshed but having had it mulling around for a while, I wanted to take some action on it. 

I have tested this in macOS 10.12, using Xcode 8.2 with Swift 3 on macOS and iOS apps.

P.S. Worth noting here that this is not reliable when it is run on a machine with 2+ monitors. The screenshot is always of the primary display (the one with the dock).

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @ohayon</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by mfurtak** [comment]
Are there tests we can be adding to ensure that this functionality stays working?

**Comment by ohayon** [comment]
Yes there are for sure. I can add some of those later on and will update the PR!

**Comment by ohayon** [comment]
Added test for macOS support https://github.com/fastlane/fastlane/pull/8468/commits/03c58143b2edc360cfb3c6f03748c735300b3d74

File: snapshot/example/ExampleMacOS/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
+//
+//  AppDelegate.swift
+//  ExampleMacOS
+//
+//  Created by Alexander Semenov on 1/19/17.
+//  Copyright © 2017 Felix Krause. All rights reserved.
```
Comment (mfurtak) [inline snapshot/example/ExampleMacOS/AppDelegate.swift]: We should figure out what to do about these copyright statements. I don't think they should stay.

**Review by mfurtak — APPROVED** [review]
Let's remove the copyright lines from the example files, but otherwise LGTM

**Comment by ohayon** [comment]
Going to follow up with another PR to remove copyright files from the project files.

**Pull request merged**